packages/blocks/azure/src/lib/actions/custom-azure-api-action.ts

```diff
@@ -0,0 +1,62 @@
+import { createCustomApiCallAction } from '@openops/blocks-common';
+import { Property } from '@openops/blocks-framework';
+import {
+  authenticateUserWithAzure,
+  azureAuth,
+  getUseHostSessionProperty,
+} from '@openops/common';
+import { runCommand } from '../azure-cli';
+import { subDropdown } from '../common-properties';
+
+const getHostAccessToken = async (
+  auth: unknown,
+  subscription?: string,
```

Comment:
why do you want the user to pass the subscription? 
does it matter for http calls token?